src/scatter_buildblock/upsample_and_fit_scatter_estimate.cxx

```diff
@@ -53,12 +54,21 @@ upsample_and_fit_scatter_estimate(ProjData& scaled_scatter_proj_data,
 {
   shared_ptr<ProjDataInfo> 
     interpolated_direct_scatter_proj_data_info_sptr(emission_proj_data.get_proj_data_info_sptr()->clone());
-  interpolated_direct_scatter_proj_data_info_sptr->reduce_segment_range(0,0);
```

Comment:
sadly this seems wrong.  your change does the reduction after the construction `interpolated_direct_scatter`.

Reply:
are you saying I need to construct it with the reduction? 
`interpolated_direct_scatter_proj_data_info_sptr(emission_proj_data.get_proj_data_info_sptr()->reduce_segment_range(0,0))`

Reply:
sorry i don't see why this is wrong though

Reply:
ok.my mistake. ignore